llvm/test/tools/UpdateTestChecks/update_test_checks/Inputs/various_ir_values.ll

```diff
@@ -1,5 +1,5 @@
 ; Just run it through opt, no passes needed.
-; RUN: opt < %s -S | FileCheck %s
+; RUN: opt < %s -S --write-experimental-debuginfo=false | FileCheck %s
```

Comment:
Is it worth adding comments to these tests explaining why new intrinsics are not used?